ref_arch/real_time_dashboards.rst

```diff
@@ -69,16 +67,13 @@ recommend :ref:`using 2-4x as many shards <faq_choose_shard_count>` as
 CPU cores in your cluster. Using this many shards lets you rebalance
 data across your cluster after adding new worker nodes.
 
-Using a replication factor of 2 means every shard is held on multiple workers. When a
-worker fails the coordinator will prevent downtime by serving queries for that worker's shards
-using the other replicas.
-
 .. NOTE::
 
-  In Citus Cloud you must use a replication factor of 1 (instead of the 2 used here). As
   Citus Cloud uses `streaming replication
   <https://www.postgresql.org/docs/current/static/warm-standby.html>`_ to achieve high
-  availability maintaining shard replicas would be redundant.
+  availability and thus maintaining shard replicas would be redundant. In a
+  production environment, you should set ``citus.shard_replication_factor`` to 2
```

Comment:
In one way of reading it, sounds like we're saying Citus Cloud is not a production environment. Maybe we could say, "in a self-hosted production environment" or "in a production environment where streaming replication is unavailable"